jenkins/shared-library/vars/defaultCiPipeline.groovy

```diff
@@ -23,13 +23,16 @@ void call(Closure body) {
 			choice name: 'BUILD_CONFIGURATION',
 				choices: ['release-private', 'release-public'],
 				description: 'build configuration'
+			choice name: 'TEST_MODE',
```

Comment:
are you planning to run coverage nightly? or why are there code coverage and non - code coverage builds.

Reply:
No, was thinking for Release builds probably dont need to run code coverage tests.   Plus if there is an issue with tests can run them without.  Can always leave it enabled if you want